uber/templates/signups/food_info.html

```diff
@@ -0,0 +1,4 @@
+{% comment %}<!--
+    Events should override this file with information about volunteer food, if applicable.
+    If this template is not overridden, then no food information will be shown to volunteers.
+-->{% endcomment %}
```

Comment:
I've made a separate PR for MagClassic at https://github.com/magfest/magclassic/pull/19/files